pkgs/development/interpreters/php/8.0.nix

```diff
@@ -42,6 +42,7 @@ base.withExtensions ({ all, ... }: with all; ([
   sockets
   soap
   sodium
+  sysvsem
```

Comment:
Looks like we may want other SysV extensions as well:

https://salsa.debian.org/php-team/php/-/blob/01284260a52f69fc24b361772bf05dab358e0f08/debian/rules.d/ext-common.mk#L18-20

Reply:
does it maybe make sense to add it to the root package? So they are enabled without adding them as extension? 🤔

Reply:
I think we prefer the ability to be able to disable extensions, where practical, to be able to have truly minimal PHP.